web/wp-content/themes/bumblebee/functions.php

```diff
@@ -192,3 +192,145 @@ function bumblebee_scripts() {
 add_image_size( 'homepage-featured-big', 385, 385, true );
 add_image_size( 'homepage-featured-small', 165, 165, true );
 add_image_size( 'grid-thumbnail', 273, 273, true );
+
+
+/**
+ * Register Custom Post Type.
+ */
+function listicle_post_type() {
+
+	$labels = array(
+		'name'               => _x( 'Listicles', 'Post Type General Name', 'listicle-post-type' ),
+		'singular_name'      => _x( 'Listicle', 'Post Type Singular Name', 'listicle-post-type' ),
+		'menu_name'          => __( 'Listicles', 'listicle-post-type' ),
+		'name_admin_bar'     => __( 'Listicle', 'listicle-post-type' ),
+		'archives'           => __( 'Listicle Archives', 'listicle-post-type' ),
+		'attributes'         => __( 'Listicle Attributes', 'listicle-post-type' ),
+		'parent_item_colon'  => __( 'Parent Item:', 'listicle-post-type' ),
+		'all_items'          => __( 'All Listicles', 'listicle-post-type' ),
+		'add_new_item'       => __( 'Add New Listicle', 'listicle-post-type' ),
+		'add_new'            => __( 'Add New', 'listicle-post-type' ),
+		'new_item'           => __( 'New Listicle', 'listicle-post-type' ),
+		'edit_item'          => __( 'Edit Listicle', 'listicle-post-type' ),
+		'update_item'        => __( 'Update Listicle', 'listicle-post-type' ),
+		'view_item'          => __( 'View Listicle', 'listicle-post-type' ),
+		'view_items'         => __( 'View Listicles', 'listicle-post-type' ),
+		'search_items'       => __( 'Search Listicle', 'listicle-post-type' ),
+		'not_found'          => __( 'Not found', 'listicle-post-type' ),
+		'not_found_in_trash' => __( 'Not found in Trash', 'listicle-post-type' ),
+	);
+
+	$rewrite = array(
+		'slug'       => 'listicle',
```

Comment:
If we change `slug` to something like `%category%`, won't this help removing the `setup_url_scheme` function?